modules/ROOT/pages/apikit-graphql-api-implementation.adoc

```diff
@@ -0,0 +1,525 @@
+                   "city": "Bolo",
+                   "zipcode": "4201"
+               }
+           }
+       ]
+   }
+}
+----
+
+=== Retrieve a Post by ID
+
+This query example requests the post data by its ID, including the post comments and the usernames.
+
+==== Make a Query
+
+----
+{
+   getPost(id: 10) {
+       id
+       user {
+           username
+       }
+       title
+       body
+       comments {
+           user {
+               username
+           }
+           body
+           replies {
+               body
+           }
+       }
+   }
+}
+----
+
+==== Trigger a Request
+
+----
+curl --location --request POST 'http://localhost:8081/graphql' \
+--header 'Content-Type: application/json' \
+--data-raw '{"query":"{getPost(id:10){id user{username}title body comments{user{username}body replies{body}}}}","variables":{}}'
+----
+
+==== Get a Response
+
+----
+{
+   "data": {
+       "getPost": {
+           "id": "10",
+           "user": {
+               "username": "bdearelli"
+           },
+           "title": "id luctus nec molestie sed justo pellentesque viverra pede ac",
+           "body": "Maecenas tristique, est et tempus semper, est quam pharetra magna, ac consequat metus sapien ut nunc. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Mauris viverra diam vitae quam. Suspendisse potenti. Nullam porttitor lacus at turpis. Donec posuere metus vitae ipsum. Aliquam non mauris.",
+           "comments": [
+               {
+                   "user": {
+                       "username": "cwalesby1p"
+                   },
+                   "body": "Praesent blandit. Nam nulla. Integer pede justo, lacinia eget, tincidunt eget, tempus vel, pede. Morbi porttitor lorem id ligula. Suspendisse ornare consequat lectus. In est risus, auctor sed, tristique in, tempus sit amet, sem.",
+                   "replies": []
+               },
+               {
+                   "user": {
+                       "username": "rricci2d"
+                   },
+                   "body": "Maecenas leo odio, condimentum id, luctus nec, molestie sed, justo. Pellentesque viverra pede ac diam. Cras pellentesque volutpat dui.",
+                   "replies": []
+               }
+           ]
+       }
+   }
+}
+----
+
+== Request Mutations Examples
+
+The *Mutation* type defines the list of mutations that the service enables.
+
+----
+...
+type Mutation {
+    # Creates a new user
+    createUser(user: UserInput!): User!
+    # Creates a new post
+    createPost(userId: ID!, title: String!, body: String!): Post!
+    # Adds a comment to a post
+    postComment(userId: ID!, postId: ID!, body: String!): Comment!
+    # Adds a reply to a comment
+    replyToComment(commentId: ID!, userId: ID!, body: String!): Comment!
+}
+...
+----
+
+=== Create a New Post
+
+This example request creates a new post and asks for some information about it.
+
+==== Make a Query
```

Comment:
"Pose a Query" seems to be a better option.
Should this title be consistent with the title in line 113?
Let's wait for the dev team comments.

Reply:
It has been decided to keep "Make a Query". Thanks!